java/example/pom.xml

```diff
@@ -11,14 +11,14 @@
     <properties>
         <app.id>com.google.appengine.examples.gcs-appengine</app.id>
         <app.version>1</app.version>
-        <appengine.version>1.9.36</appengine.version>
+        <appengine.version>1.9.64</appengine.version>
```

Comment:
is an internal version of this library usually updated outside of updating this repo?

Reply:
I'm not sure I follow. Can you elaborate? Do you mean something in google3?

This PR is purely about updating the example code.

Reply:
I misread the path, thanks for clarifying.